backend/src/packages/users/user.repository.ts

```diff
@@ -11,6 +11,14 @@ class UserRepository
     this.userModel = userModel;
   }
 
+  public async find(email: string): ReturnType<IRepository['find']> {
```

Comment:
```suggestion
  public async find(email: string): Entity {
```
should be user entity here